src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs

```diff
@@ -43,7 +43,7 @@ public ActiveDirectoryConfiguration ActiveDirectoryConfiguration
         private string _roleAdminGroupDN;
         
         /// <summary>
-        /// Optional. Gets or sets the Distinguished Name for role
+        /// Optional. Optional. Gets or sets the Distinguished Name for role
```

Comment:
I need to add a couple of version files to this PR. Could you please hold on the merge while I update the PR?